examples/inverse/plot_psf_ctf_vertices_lcmv.py

```diff
@@ -0,0 +1,133 @@
+fname_cov = data_path + '/MEG/sample/sample_audvis-cov.fif'
+fname_evo = data_path + '/MEG/sample/sample_audvis-ave.fif'
+raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
+
+# Read raw data
+raw = mne.io.read_raw_fif(raw_fname, preload=True)
+
+# only pick good EEG/MEG sensors
+raw.info['bads'] += ['EEG 053']  # bads + 1 more
+picks = mne.pick_types(raw.info, meg=True, eeg=True, exclude='bads')
+
+# Find events
+events = mne.find_events(raw)
+
+# event_id = {'aud/l': 1, 'aud/r': 2, 'vis/l': 3, 'vis/r': 4}
+event_id = {'vis/l': 3, 'vis/r': 4}
+
+tmin, tmax = -.2, .25  # epoch duration
+epochs = mne.Epochs(raw, events, event_id=event_id, tmin=tmin, tmax=tmax,
+                    picks=picks, baseline=(-.2, 0.), preload=True)
+
+# covariance matrix for pre-stimulus interval
+tmin, tmax = -.2, 0.
+cov_pre = mne.compute_covariance(epochs, tmin=tmin, tmax=tmax,
+                                 method='empirical')
+
+# covariance matrix for post-stimulus interval (around main evoked responses)
+tmin, tmax = 0.05, .25
+cov_post = mne.compute_covariance(epochs, tmin=tmin, tmax=tmax,
+                                  method='empirical')
+
+# read forward solution
+forward = mne.read_forward_solution(fname_fwd)
+# use forward operator with fixed source orientations
+forward = mne.convert_forward_solution(forward, surf_ori=True,
+                                       force_fixed=True)
+
+# read noise covariance matrix
+noise_cov = mne.read_cov(fname_cov)
+
+# get valid measurement info
+raw = raw.pick_types(meg=True, eeg=True, exclude='bads')
+info = raw.info
+
+# regularize noise covariance (we used 'empirical' above)
+noise_cov = mne.cov.regularize(noise_cov, info, mag=0.1, grad=0.1,
+                               eeg=0.1, rank='info')
+
+##############################################################################
+# Compute LCMV filters with different data covariance matrices
+# ------------------------------------------------------------
+
+# compute LCMV beamformer filters for pre-stimulus interval
+filters_pre = make_lcmv(info, forward, cov_pre, reg=0.05,
+                        noise_cov=noise_cov,
+                        pick_ori=None, rank=None,
+                        weight_norm=None,
+                        reduce_rank=False,
+                        verbose=False)
+
+# compute LCMV beamformer filters for post-stimulus interval
+filters_post = make_lcmv(info, forward, cov_post, reg=0.05,
+                         noise_cov=noise_cov,
+                         pick_ori=None, rank=None,
+                         weight_norm=None,
+                         reduce_rank=False,
+                         verbose=False)
+
+##############################################################################
+# Compute resolution matrices for LCMV beamformers with different data
+# covariance matrices
+# --------------------------------------------------------------------
```

Comment:
this does not render well. See

https://17962-1301584-gh.circle-artifacts.com/0/dev/auto_examples/inverse/plot_psf_ctf_vertices_lcmv.html#sphx-glr-auto-examples-inverse-plot-psf-ctf-vertices-lcmv-py

Reply:
This one appears not to have been fixed yet

https://18207-1301584-gh.circle-artifacts.com/0/dev/auto_examples/inverse/plot_psf_ctf_vertices_lcmv.html#compute-lcmv-filters-with-different-data-covariance-matrices

Reply:
Can you please highlight what to look for? Alex' link looked fine to me, in yours I see double...